docs/diagrams/LessonClassDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+package Lesson <<Rectangle>>{
+Class Lesson
+Class Capacity
+Class LessonName
+Class Price
+Class Students
+Class Timing
+Class Student
+}
+
+Class HiddenOutside #FFFFFF
+HiddenOutside --> Lesson
+
+Lesson *--> "1" Capacity
```

Comment:
I was wondering if the multiplicities have to be `0..1` for the optional parameters?

Reply:
Technically we have guarantees that the fields are all present and non-null so I think this is correct - even though the user can omit these fields, the objects themselves will have them

Reply:
Yes I agree with @wpinrui! Even if the optional parameters were not given by the user during the adding of the lesson, they are still instantiated with an empty string, and so they will not be `null`

Reply:
Right thanks for clarifying!